### ISSUE TITLE
Openni Launch Doesn't work at times after upgrade

### ISSUE THREAD

**Issue by ksatyaki** [nh2/asus-xtion-fix#1]
At certain times, the openni launch doesn't recognize the device. That is, under normal circumstances there is: "Device "1d27/0601@2/4" with serial number "1206070219" connected". This doesn't show up at certain times. DMESG output when this happens is: 
[ 2337.622523] 3:2:1: usb_set_interface failed (-110)
[ 2342.624647] 3:2:1: usb_set_interface failed (-110)
[ 2347.626761] 3:2:1: usb_set_interface failed (-110)
[ 2352.628837] 3:2:1: usb_set_interface failed (-110)
[ 2357.630951] 3:2:1: usb_set_interface failed (-110)
[ 2362.633276] 3:2:1: usb_set_interface failed (-110)
[ 2367.635383] 3:2:1: usb_set_interface failed (-110)
[ 2372.645469] 3:2:1: usb_set_interface failed (-110)
[ 2377.647565] 3:2:1: usb_set_interface failed (-110)
[ 2382.649661] 3:2:1: usb_set_interface failed (-110)
[ 2387.651737] 3:2:1: usb_set_interface failed (-110)

So many such lines.

After unplugging and re-plugging the USB cable, and restarting the openni launch, everything works fine.

**Comment by manel-ferreira-01** [comment]
Did you eventually found a solution to this?

**Comment by ksatyaki** [comment]
I don't think so. I guess it was a bad combo of the drivers at the time and the device itself. I haven't used the device in about 7 years.
